### DISCUSSION TITLE
Ignore prerelease notifications

### DISCUSSION THREAD

**Discussion by samfundev** [community/community#29192] [Other Feature Feedback, Questions, & Ideas]
When watching a repository it's only possible to get notifications for any release, including prereleases. Some repositories use prereleases as a nightly build which means getting notified everyday. I would like to get notified only when a stable release happens, so that I can see what's changed in the latest stable version.

**Comment by GabyCT** [comment]
Please take a look at https://docs.github.com/es/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#about-participating-and-watching-notifications

**Reply by samfundev** [reply]
Sorry, I don't understand how this is supposed to help me. I already selected to only receive notifications for releases, but I cannot exclude preleases.
![image](https://user-images.githubusercontent.com/6759716/184228059-d7258a13-d572-4e1a-b3d0-f17c18fcf85f.png)

**Comment by hugo082** [comment]
Would love to be able to filter by release version, e.g. being able to get notifications only for major or minor releases, not patches

**Comment by 0xpr03** [comment]
Same here, please allow me to ignore pre-release, or even better: blacklist certain names.

I don't want "nightly" release notifications, or "rc.2". That's the point I may as well ignore the releases at all.

**Comment by AMDmi3** [comment]
Seconded. There are projects which releases are critical to me, but these also do nighly builds which are published as (pre-)releases:

https://github.com/widelands/widelands/releases
https://github.com/buddhi1980/mandelbulber2/releases

Watching releases for these projects via watch->custom->releases knob as on the screenshot above makes by feed polluted by prereleases which I don't need, making the whole watch feature useless. So it would be nice to have an option to exclude prereleases from the watch.

**Comment by u3u** [comment]
Totally agree, I just want to be notified of the latest release of `next.js`: https://github.com/vercel/next.js/releases

<img width="320" alt="image" src="https://user-images.githubusercontent.com/20062482/195846368-12830017-9aff-4f89-a8c4-4186566681b6.png">

We need the "Latest releases only" option

**Comment by josStorer** [comment]
Totally agree, also I think it would be better to provide `Watch All Activity But Ignore Pre-release` in the watch drop-down list, especially for tiny repositories that many people will watch all activity but not hope to get emails of pre-release

**Comment by t0ma5** [comment]
One more for **ignore pre-releases**... I get too many useless notifications :(

**Comment by tristan-f-r** [comment]
I use **automated** releases for data collection and it also causes clutter on my notifications & feed for myself and my friends:

![image](https://user-images.githubusercontent.com/26509014/218472961-22b83fdb-450e-4244-805f-f3fbef0882e9.png)

**Comment by mh4ckt3mh4ckt1c4s** [comment]
Same here, I want to watch the stable releases of [podman-desktop](https://github.com/containers/podman-desktop/) but they do a prerelease for each new commit which make the amount of notifications useless and boring.

**Comment by AndreCunha1** [comment]
Seems like a duplicate of #4993, but anyway I am upvoting and subscribing to both just in case 😄

**Comment by samfundev** [comment]
This definitely does seem like a duplicate, closing in favor of #4993. If you haven't, go upvote that discussion.
